plugins/cloudbuild/src/components/useWorkflowRuns.ts

```diff
@@ -33,8 +33,11 @@ export type WorkflowRun = {
   rerun: () => void;
 };
 
-export function useWorkflowRuns(options: { projectId: string }) {
-  const { projectId } = options;
+export function useWorkflowRuns(options: {
+  projectId: string;
+  cloudBuildFilter: string;
```

Comment:
you could make it optional so that we can ship the changes as `patch`

```suggestion
  cloudBuildFilter?: string;
```

Reply:
If the value does not exist, then the API query will fail, so it must be populated. This is why the function evaluates whether or not the annotation exists and if it does not, then it defaults to the entity.name:

https://github.com/backstage/backstage/pull/21397/files/568121143a10db1a54ca6a394be49522de3fd206#diff-b9357e33db3863fcb9d0fb872b2bf0069d2d1fd18da600b7cca6790782a4b7f1R27